lib/game.rb

```diff
@@ -1,17 +1,63 @@
 require_relative "board.rb"
+require_relative "coordinates.rb"
 
 class Game
+  def initialize(std_out, std_in)
+    @std_out = std_out
+    @std_in = std_in
+  end
+
   def self.run(*args)
-    Game.new.run(*args)
+    Game.new(*args).run
   end
 
-  def run(output)
-    output.puts empty_board
+  def run
+    coordinates = start_game
```

Comment:
One of the design guides is
> Don't use local variables, make methods instead that capture that logic (and memoize if necessary)

Perhaps a method further down can update a `@coordinates` variable?

Reply:
I like that idea, what do you think about 04146cb4ca666e71809175cd780a7e45b0f20859